spec/integration/status_updates_spec.rb

```diff
@@ -1,69 +1,54 @@
 RSpec.describe "Receiving a status update", type: :request do
-  let(:delivery_attempt) { create(:delivery_attempt, status: "sent") }
-  let(:reference) { delivery_attempt.id }
-
-  let(:permissions) { %w[signin status_updates] }
-  let(:user) { create(:user, permissions: permissions) }
-  before { login_as(user) }
-
   describe "#create" do
```

Comment:
Out-of-scope: is there a reason why this directory is called "integration", instead of "requests"? The latter is more specific than the former, and seems to capture the current contents.

Reply:
It seems to be part of a broader testing strategy that didn't really take off: https://github.com/alphagov/email-alert-api/pull/300. I can see that spec/requests was renamed to spec/integration. I'd be happy for us to change it back.

Reply:
Hmm...interesting. Looking again I forgot this repo also has `spec/features`, which are also `type: :request`. It also turns out there are [non-request specs](https://github.com/alphagov/email-alert-api/blob/master/spec/integration/anonymise_email_addresses_spec.rb) in this directory, and [a README](https://github.com/alphagov/email-alert-api/blob/master/spec/integration/README.md) that conflicts with the tighter scope I was suggesting. While it's a shame, I think it would introduce more confusion to rename it; if it starts causing pain, then we should consider revising the test strategy. Thanks for the chat about it!